src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -48,14 +54,26 @@ export default function WebexMeeting({className, meetingID, style}) {
 
   return (
     <div className={cssClasses} style={style}>
-      {meetingDisplay}
-      {showSettings && (
-        <Modal
-          onClose={() => adapter.meetingsAdapter.toggleSettings(ID)}
-          title="Settings"
-        >
-          <WebexSettings meetingID={ID} />
-        </Modal>
+      <div className="webex-meeting__meeting">
+        {meetingDisplay}
+        {showSettings && (
```

Comment:
settings modal should only be shown when one of InMeeting or InterstitialMeeting is shown. So please add it to meetingDisplay.

same for roster

same for controls

the controls don't need a condition anymore, because the `meetingDisplay` has separate cases for no state or left meeting.

Reply:
done